strapi/strapi
# [fix]: use explicit imports for fontawesome in order to decrease bundle size.

### What does it do?

Found this discuttion about the size of the bundle. 

https://github.com/FortAwesome/react-fontawesome/issues/70

We need to import expliciting the component want in order of the webback be able to  only import what you need. 

Following issue recomendation from FortAwesome github, issue 70. https://github.com/FortAwesome/react-fontawesome/issues/70

Just changed the way the components are imported: 
from
`
import { faQuestion,faTimes } from '@fortawesome/free-solid-svg-icons/'`

to 

```
import { faQuestion } from '@fortawesome/free-solid-svg-icons/faQuestion';
import { faTimes } from '@fortawesome/free-solid-svg-icons/faTimes';
```

Look at the icons that were "changed"
![image](https://user-images.githubusercontent.com/6992588/194719637-b78b2cd9-0d3c-4e8b-8cbc-b1a3e502c19f.png)


### Why is it needed?

Make bundlesize smaller. 

### How to test it?

run yarn analyze:bundle in packages/core/admin
see report and investigate yourself.


before:
![image](https://user-images.githubusercontent.com/6992588/194719769-21d5586b-b232-44fc-9085-7deefe2d13dd.png)


after:
![image](https://user-images.githubusercontent.com/6992588/194719719-196f3eec-96da-4387-9061-ebb572f75a1e.png)



result: 

4.34 MB to  642.77 KB  -> 93% do decrese of this package
### Related issue(s)/PR(s)

Fixes #14572
